proxy/core/src/main/scala/io/cloudstate/proxy/ProtobufGeneratedMessageRegisterFeature.scala

```diff
@@ -0,0 +1,33 @@
+package io.cloudstate.proxy
+
+import com.oracle.svm.core.annotate.AutomaticFeature
+import org.graalvm.nativeimage.hosted.Feature.QueryReachabilityAccess
+import org.graalvm.nativeimage.hosted.{Feature, RuntimeReflection}
+
+import scala.collection.JavaConverters._
+
+@AutomaticFeature
+final class ProtobufGeneratedMessageRegisterFeature extends Feature {
+
+  override def duringAnalysis(access: Feature.DuringAnalysisAccess) =
+    List(
+      "akka.protobuf.GeneratedMessage",
+      "akka.protobuf.GeneratedMessage$Builder",
+      "akka.protobuf.ProtocolMessageEnum",
+      "com.google.protobuf.GeneratedMessageV3",
+      "com.google.protobuf.GeneratedMessageV3$Builder",
+      "com.google.protobuf.ProtocolMessageEnum",
+      "scalapb.GeneratedMessage"
+    ).foreach { className =>
+      val cls = access.findClassByName(className)
```

Comment:
Wouldn't we want to error if we cannot find these classes?

Reply:
I considered it.  There's two situations where that could happen (as I see it):

1. `scalapb.GeneratedMessage` (as an example) _is_, legitimately, on the classpath, but `access.findClassByName` doesn't see it
2. `scalapb.GeneratedMessage` is not on the classpath

If (2) is the case, then throwing an error is wrong and an annoyance.  If (1) is the case, then would it mean Graal is broken?  Would it take long for us to connect the dots back to the skip here?  🤷  I'm thinking we would, so I went with this impl.

Reply:
@dwijnand I think the right solution is to do what native-image does—it logs if it cannot find the corresponding class for the configuration.

Reply:
Yeah, I didn't find a logger anywhere.

If I find one I'll come back and add it here.